src/Radicle/Daemon/Common.hs

```diff
@@ -26,18 +27,19 @@ data Polling = HighFreq Int64 | LowFreq
 instance A.ToJSON ReaderOrWriter
 instance A.FromJSON ReaderOrWriter
 
-data Chain id idx subs = Chain
-    { chainName         :: id
-    , chainState        :: Bindings (PrimFns Identity)
-    , chainEvalPairs    :: Seq.Seq (Value, Value)
-    , chainLastIndex    :: Maybe idx
-    , chainMode         :: ReaderOrWriter
-    , chainSubscription :: subs
-    , chainLastUpdated  :: Time.SystemTime
-    , chainPolling      :: Polling
+data Machine = Machine
+    { machineId           :: MachineId
+    , machineState        :: Bindings (PrimFns Identity)
+    , machineLastIndex    :: Maybe MachineEntryIndex
+    , machineMode         :: ReaderOrWriter
+    , machineSubscription :: TopicSubscription
+    , machineLastUpdated  :: Time.SystemTime
+    , machinePolling      :: Polling
     } deriving (Generic)
 
-newtype Chains id idx subs = Chains { getChains :: CMap.CMap id (Chain id idx subs) }
+data CachedMachine = UninitialisedReader Time.SystemTime | Cached Machine
```

Comment:
Can you put a comment saying what this `SystemTime` is for/about?

Reply:
:+1: